app/components/forms/user-profile-form.js

```diff
@@ -44,15 +44,16 @@ export default Component.extend(FormMixin, {
         },
         phone: {
           identifier : 'phone',
+          optional   : true,
           rules      : [
             {
               type   : 'empty',
```

Comment:
remove the empty rule completely, optional and empty both don't make sense.

Reply:
ok